model/bin/ww3_from_ftp.sh

```diff
@@ -10,19 +10,32 @@ curr_dir=`pwd`
 # Set WW3 code version
 ww3ver=v7.12.6
 
+interactive='y'
+if [ $# -eq 3 ] ; then
+  interactive='n'
+  ww3dir=$1  # [ '../../']
```

Comment:
To be more consistent with other scripts should we update this to be pointing to the model directory instead of the top level directory?